ground/uavobjgenerator/uavobjgenerator.pro

```diff
@@ -5,14 +5,14 @@ QT += xml
 QT -= gui
 
 macx {
-    QMAKE_CFLAGS_X86_64 += -mmacosx-version-min=10.7
-    QMAKE_CXXFLAGS_X86_64 = $$QMAKE_CFLAGS_X86_64
+    QMAKE_MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
It's consistent with the other GCS makefiles; I use a lambda to sort which is a C++11 feature.  C++11 in general was brought in with PR #1738.  

(the deployment target change is necessary to have a valid C++11 standard library).